src/components/NewBook.jsx

```diff
@@ -0,0 +1,9 @@
+import React from 'react';
+
+function NewBook() {
+  return (
+    <div>NewBook</div>
+  );
+}
+
+export default NewBook;
```

Comment:
- [OPTIONAL] I put this like OPTIONAL because the last CR forgot it, but please complete all the project requirements, this component should return a form at the bottom of the page 👇🏽 

![image](https://github.com/alvaroiporre/Bookstore/assets/58642949/10ada5d8-930f-47cb-bef8-d2f85e6f9107)